### PR TITLE
feat: add "Mouse" command for mouse support

### PR DESCRIPTION
**Pull request by XMinty7**
The added "Mouse" command takes x and y coordinates within a 320x180 input space and moves the mouse accordingly. To avoid desyncs, the command only runs on window sizes that are integer upscales of the game's original 320x180 size.
Edit: Now supports clicking/holding left, middle and right mouse buttons.

### CODE REVIEW

File: CelesteTAS-EverestInterop/Source/TAS/Input/Commands/MouseCommand.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    public static Point? Position { get; private set; } = null;
+
+    // "Mouse, X, Y",
+    [TasCommand("Mouse")]
+    private static void Press(string[] args) {
+        if (args.IsEmpty()) {
+            return;
+        }
+
+        if (!int.TryParse(args[0], out int x) || x > 320 || x < 0) {
```
Comment (Kalobi) [inline CelesteTAS-EverestInterop/Source/TAS/Input/Commands/MouseCommand.cs:20]: Surely in a window of width 320, the valid values for inside the window would either range from 0 to 319 or 1 to 320, not 0 to 320.

**Review by Kalobi — COMMENTED** [review]
This is somewhat pointless (at least for the applications I know of, like Guneline) without a way to send mouse clicks.

File: CelesteTAS-EverestInterop/Source/TAS/Input/Commands/MouseCommand.cs (hunk 1 above)
Comment (XMinty7) [reply CelesteTAS-EverestInterop/Source/TAS/Input/Commands/MouseCommand.cs]: Modified as appropriate.

**Comment by DemoJameson** [comment]
I am currently on vacation, when I have time I will review and merge

**Review by Kalobi — COMMENTED** [review]
Pressing mouse buttons doesn't work for me, even though setting the position does.

**Comment by DemoJameson** [comment]
@Kalobi can you give it a try

**Comment by Kalobi** [comment]
@DemoJameson Still having the same issue.

**Comment by Kalobi** [comment]
@DemoJameson Now it's not even setting the position.

**Comment by DemoJameson** [comment]
How do you test it

It might be Mouse.SetPosition is no longer called, I only set the MInput.Mouse.CurrentState

I tested the Chinese New Year Collab map, the right click shows the wheel but it needs to move before clicking, the left click is normal, I think that is a problem that the New Year Helper needs to solve

**Comment by DemoJameson** [comment]
ok, we can test with debug the map, here is what I tested
```
unsafe
console load 2
   1
console e
   1
console load 1
   1
console e
   1

#Start
Mouse 0 20
   1
Mouse 319 179
   1
Mouse M 170 80
   1
Mouse M 100 100
   1
Mouse 100 100 10
   1
Mouse L 120 100
   1
Mouse L 120 110
   1
Mouse R 120 110
   1
···

**Comment by Kalobi** [comment]
I tested with Guneline.

**Comment by Kalobi** [comment]
> ok, we can test with debug map, here is what I tested
This seems to be working, so it's probably a Guneline issue.

**Pull request merged**